data.sql

```diff
@@ -50,3 +50,43 @@ SET owner_id =
     WHEN name IN ('Charmander', 'Squirtle', 'Blossom') THEN (SELECT id FROM owners WHERE full_name = 'Melody Pond')
     WHEN name IN ('Angemon', 'Boarmon') THEN (SELECT id FROM owners WHERE full_name = 'Dean Winchester')
 END;
+
+-- Insert the following data into the vets table
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('William Tatcher', 45, '2000-04-23'),
+       ('Maisy Smith', 26, '2019-01-17'),
+       ('Stephanie Mendez', 64, '1981-05-04'),
+       ('Jack Harkness', 38, '2008-06-08');
+
+-- Insert the following data into the specializations table
+
+INSERT INTO specializations (vet_id, species_id)
+VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon')),
+       ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Digimon')),
+       ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Pokemon')),
+       ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));

```

Comment:
* I get this error when I tried to run these queries Kindly fix them. HINT ( you can try adding the `id` number for each vet and species  example: `william tatcher: 1`, `pokemon: 1`  ) you can get this `ids` from the qureies used to insert data into the species table and vets table
![image](https://user-images.githubusercontent.com/52497246/213903275-1ecff312-12bf-44eb-a6a3-3cfc43da18bf.png)